actions/workflows/st2_prep_release.yaml

```diff
@@ -125,6 +125,16 @@ st2cd.st2_prep_release:
                 local_repo: <% 'st2_dockerfiles_' + $.local_repo_sfx %>
                 hosts: <% $.host %>
                 cwd: <% $.cwd %>
+            on-success:
+                - wait_for_community_docker_images
+        wait_for_community_docker_image:
+            action: st2cd.wait_for_docker_image
+            input:
+                org: <% $.fork %>
+                image: st2workflowengine
+                version: <% $.version %>
+                tries: 45
```

Comment:
Mistral has built-in `retry`: `delay` mechanism which would be nicer to use: https://docs.openstack.org/mistral/ocata/main_features.html#yaml-example
But unsure about Orquesta and so I'm find with current approach unless if it does the job :+1: